docs/src/js/classes/MatchQuery.md

```diff
@@ -37,14 +35,17 @@ Creates an instance of MatchQuery.
 * **column**: `string`
     The name of the column to search within.
 
-* **boost**: `number` = `1.0`
-    (Optional) The boost factor to influence the relevance score of this query. Default is `1.0`.
+* **options?**
+    Optional parameters for the match query.
+    - `boost`: The boost factor for the query (default is 1.0).
```

Comment:
is the boost factor only negative boost factor?
searched over the network, this explaination is more clear "A boosting query returns documents that match a positive query. Among those documents, the ones that also match the negative query are scored lower in relevance (their relevance score is multiplied by the negative boosting factor)."